src/corehost/cli/deps_format.cpp

```diff
@@ -68,7 +63,7 @@ void deps_json_t::reconcile_libraries_with_targets(
                 size_t pos = library.first.find(_X("/"));
                 entry.library_name = library.first.substr(0, pos);
                 entry.library_version = library.first.substr(pos + 1);
-                entry.library_type = _X("package");
+                entry.library_type = pal::to_lower(library.second.at(_X("type")).as_string());
```

Comment:
As far as I know there is only `project` and `package` for this type field. The only other "type" we have for this kind of entry is a framework reference assembly which only applies to Desktop CLR